packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -30,16 +32,21 @@ export default /* @ngInject */ ($stateProvider) => {
       goToMigrateConfirm: /* @ngInject */ (
         $state,
         atInternet,
-        migrationTrackingPrefix,
-      ) => () => {
+        vpsMigration,
+      ) => (selectedPlan) => {
         atInternet.trackClick({
-          name: `${migrationTrackingPrefix}::confirm`,
+          name: `vps::vps-migration::from_${vpsMigration.plans[0].currentPlan}_to_${selectedPlan.planCode}::confirm`,
```

Comment:
**suggestion:** Instead of re-creating `vps::vps-migration::from_${vpsMigration.plans[0].currentPlan}`, `migrationTrackingPrefix` can be used.

Reply:
Sure Anoop will add as suggested Thank you